src/actions/TableActions.js

```diff
@@ -419,6 +419,11 @@ const isActiveWifChanged = (privateKey, basePrivateKeys, permissionForm) => {
 			return acc;
 		}, {});
 
+	const addedPubWithoutWif = Object.keys(newPrivateKeys)
+		.filter((k) => !oldPrivateKeys[k] && !newPrivateKeys[k].value);
+	for (let i = 0; i < addedPubWithoutWif.length; i += 1) {
+		delete newPrivateKeys[addedPubWithoutWif[i]];
```

Comment:
better create new object with required props ( for + delete is not good decision)